chain-core/src/common/merkle_tree.rs

```diff
@@ -190,7 +160,7 @@ impl Path {
 }
 
 /// Inclusion proof of a value
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq, Clone, Encode, Decode)]
```

Comment:
perhaps a TODO note (that was in the original) that the encoding may not be that efficient, so a custom one with BitVec+vec of hashes could be considered